pkgs/build-support/rust/hooks/default.nix

```diff
@@ -70,13 +70,17 @@ in {
 
         # Target platform
         rustTarget = ''
-          [target."${rust.toRustTarget stdenv.buildPlatform}"]
+          [host."${rust.toRustTarget stdenv.buildPlatform}"]
```

Comment:
I think both work

Reply:
I'd have a mild preference for the former, just because it means we don't have to worry about what happens if e.g. using a custom build spec.  But I don't think it's too important, and would understand if you wanted it to be consistent with what we were already doing for `target`.

Reply:
done